package.json

```diff
@@ -22,11 +22,19 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "@emotion/css": "11.1.3",
-    "react-select": "5.2.1"
+    "react-select": "5.2.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "@grafana/data": "canary",
```

Comment:
Would it make sense to point to `latest` now?

Reply:
Hmm... `latest` might work but generally with peerDependencies they set a semver range. I think something like `^9.2.0` should work here for `@grafana/` packages. It'll also help prevent noise in projects that consume this package.